docs/faq/migration/asset-migration.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_label: 'Asset Migration'
+sidebar_position: 2
+---
+
+# Asset Migration
+
+### Will previous Universal Profiles be migrated to mainnet?
+
+LUKSO will migrate all Universal Profiles previously created on the L14 Testnet by [The Dematerialised Marketplace](https://thedematerialised.com/) or [Universal Profile Explorer](https://universalprofile.cloud/). If you deployed custom profiles on older testnets, you would be responsible for migrating them to the mainnet.
+
+### Will previously aquired NFTs be migrated to mainnet?
+
+LUKSO will migrate all previously issued NFTs by the [LUKSO](https://lukso.network/) and [The Dematerialised](https://thedematerialised.com/) brands. If you deployed custom assets and services on older testnets, you would be responsible for migrating them to the mainnet.
```

Comment:
there are more link Out of the Ether NFT, and other ones.

Reply:
Yes, but OOE was done in collaboration with LUKSO. I will update both, to say we will migrate those with official collaborations

Reply:
ok